Sources/_StringProcessing/PrintAsPattern.swift

```diff
@@ -163,23 +177,42 @@ extension PrettyPrinter {
         terminateLine()
       }
       
-      switch child {
-      case let .customCharacterClass(ccc):
-        if ccc.isSimplePrint {
-          printSimpleCCC(ccc)
-          return
-        }
-        
-        break
-      case let .convertedRegexLiteral(.customCharacterClass(ccc), _):
-        if ccc.isSimplePrint {
-          printSimpleCCC(ccc)
-          return
+      // We can only do this for Optionally, ZeroOrMore, and OneOrMore. Cannot
+      // do it right now for Repeat.
```

Comment:
What's going on with `Repeat`? Does it not have the right one-`RegexComponent` inits for use with this?

Reply:
Oh, it looks like it does. I was trying to use it like `Repeat(count: 2, ...)` but it seems its `Repeat(..., count: 2)` (which reads better now that I'm thinking about it.)

Reply:
Yeah, it's that weird pattern where the closure-based version has to move the "thing to repeat" to the end, but it otherwise comes first.

Reply:
going to merge as is and will fix this in another PR